peach/src/main/java/com/hartwig/hmftools/peach/data_loader/HaplotypeEventLoader.java

```diff
@@ -0,0 +1,85 @@
+package com.hartwig.hmftools.peach.data_loader;
+
+import com.hartwig.hmftools.common.genome.chromosome.Chromosome;
+import com.hartwig.hmftools.common.genome.chromosome.HumanChromosome;
+import com.hartwig.hmftools.peach.event.VariantHaplotypeEvent;
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.readers.LineIterator;
+import htsjdk.variant.variantcontext.GenotypeType;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFCodec;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static com.hartwig.hmftools.peach.PeachUtils.PCH_LOGGER;
+import static htsjdk.tribble.AbstractFeatureReader.getFeatureReader;
+
+public class HaplotypeEventLoader
+{
+    public static Map<String, Integer> loadRelevantVariantHaplotypeEvents(
+            String vcf, String sampleName, Map<Chromosome, Set<Integer>> relevantVariantPositions
+    )
+    {
+        Map<String, Integer> eventIdToCount = new HashMap<>();
+        try(
+                AbstractFeatureReader<VariantContext, LineIterator> reader = getFeatureReader(
+                        vcf, new VCFCodec(), false)
+        )
+        {
+            for(VariantContext variantContext : reader.iterator())
+            {
+                if(variantContext.isFiltered())
+                    continue;
```

Comment:
using continues this way can lead to confusion and occasionally bugs.  I would prefer to just use brackets and either nested or combined clauses in if statements.